src/Resources/config/doctrine/BaseMedia.orm.xml

```diff
@@ -14,7 +14,7 @@
         </field>
         <field name="width" column="width" type="integer" nullable="true"/>
         <field name="height" column="height" type="integer" nullable="true"/>
-        <field name="length" column="length" type="decimal" nullable="true"/>
+        <field name="length" column="length" type="decimal" nullable="true" precision="10"/>
```

Comment:
https://www.doctrine-project.org/projects/doctrine-dbal/en/4.0/reference/schema-representation.html#portable-options

says

> Defaults to 10.

Still going to verify this

Reply:
Schema create SQL

ORM 2 without specified `precision`: `length NUMERIC(10, 0) DEFAULT NULL` 
ORM 3 with specified `precision = 10`: `length NUMERIC(10, 0) DEFAULT NULL`

 So this change should keep BC